### ISSUE TITLE
Creative Exporter seems rate limited with fluids

### ISSUE THREAD

**Issue by amybeth269** [starforcraft/Cable-Tiers#1]
I have a Creative Exporter (set to Fluid mode) setup to export Brine into a Tank (from Tank Null) from main RS system.  There's plenty of Brine in the RS system, but the Tank is filling slowly (0.2-0.3 kB/sec or so).  Still faster than the baseline stuff though :)

**Comment by amybeth269** [comment]
I did some comparative testing just to give any other info I could & Elite seems to do 0.1 mB w Stack and 2 Speed, but Ultra doesn't seem to send any fluids at all even with 4 Speeds.

**Comment by starforcraft** [comment]
hmm... okay thats weird I will look into that :)

**Comment by starforcraft** [comment]
hmm... weird it seems to work for me https://streamable.com/gdio87

**Comment by amybeth269** [comment]
Odd, I wonder if it's an issue on my end then.  I haven't changed any default settings of anything though.  ATM 6 current version, with your mod added in.  It does seem faster on this go 'round, though it still doesn't send all of it - https://streamable.com/9d2ref

**Comment by starforcraft** [comment]
Can you try the Creative Exporter on an Dynamic Tank maybe?

**Comment by amybeth269** [comment]
Good call - tested with the Dynamic Tank and absolutely no issues zippin' stuff back & forth.  

Did a few more tests with the Tank Null stuff too while I was fiddling with things.  The Creative Importer works fine & is instant from the docked Tanks.  I also tested with fluids that weren't incrementing/decrementing in my Refined Storage system to see if that was a component of the problem, but using a fluid w 2 billion available that wasn't increasing or decreasing still lead to chunks moving at a time (in differing amounts 64k one time, 640k another, 320k; sort of jumped all over).  Tested a Tank 4 since that's what you had in your example and it was doing the "bit at a time" thing too.  

Hope any of that is helpful!  It's such a useful bit of work you've done.  And thanks for suggesting the Dynamic Tanks, I'll likely switch over to them for both ease & aesthetics
